neptune-client-api.md

```diff
@@ -0,0 +1,56 @@
+
+
+```python
+from neptune.session import Session
+
+session = Session("token=")
+```
+
+
+```python
+project = session.get_project("neptune-ml/Ships")
+```
+
+
+```python
+exp = project.create_experiment( # create experiment, with params and tags
+    params={
+    "lr__C": 1000,
+    "epochs": 10,
+    "shuffle": true
+}, 
+    tags=["solution-6", "stacking"],
+    send_hardware_metrics=False, # default = True
+    run_monitoring_thread=False, # default = True
+    upload_sources_files=["main.py"], # default = this file
+    handle_uncaught_exceptions=False # default = True
+)
+```
+
+
+```python
+exp.send_metric("ROC_AUC", 0.75)
+exp.send_metric("ROC_AUC_STD", 0.002)
+exp.send_text("log", "done")
+exp.send_image("false", "/tmp/failed_class.png")
+```
+
+
+```python
+exp.set_tags(exp.get_tags().append("new-tag")) # update tags
```

Comment:
@lukasz-walkiewicz Could we go even simpler here?

```python
exp.tags.append('new-tag')
```

Reply:
Yes, we could but I think it would be a bit misleading for the user that `append` method makes a HTTP request to Neptune under the hood. Furthermore, we would have to implement the rest of list manipulating methods such as: `sort` and `reverse` which, IMO, makes little sense.